src/software/backend/replay_logging/replay_logger.h

```diff
@@ -29,12 +29,12 @@ class ReplayLogger : public FirstInFirstOutThreadedObserver<SensorMsg>
 
    private:
     /**
-     * Adds a SensorMsg to the current chunk. This will also save it to disk and
+     * Adds a SensorProto to the current chunk. This will also save it to disk and
      * clear the chunk in memory.
      *
-     * @param frame a SensorMsg
+     * @param frame a SensorProto
```

Comment:
Why is the `@param` `frame` and not `msg`?

Reply:
@jordanzliu can probably explain better but I think he's thinking about replay like a video, and so each SensorProto is like a frame of video being captured. Very poetic